kalitectl.py

```diff
@@ -571,6 +572,24 @@ def start(debug=False, watch=False, daemonize=True, args=[], skip_job_scheduler=
             'server.socket_port': port
         })
 
+    # idealy we'd have a setting element, but these are imported much later
+    # and it will require a much deeper change
+    idle_timeout = int(os.getenv("KALITE_IDLE_TIMEOUT", 0))
+    if idle_timeout:
+        cherrypy.engine.last_busy = time.time()
+        def check_connections():
+            for thread in threading.enumerate():
+                if 'worker_busy' in dir(thread) and thread.worker_busy.isSet():
+                    cherrypy.engine.last_busy = time.time()
+            idle = time.time() - cherrypy.engine.last_busy
+            if (idle > idle_timeout):
+                print ("Idle for %s, exiting" % idle_timeout)
+                # we need to do both to cleanly exit
+                cherrypy.engine.exit()
+                stop()
+
+        cherrypy.process.plugins.Monitor(cherrypy.engine, check_connections, frequency=(idle_timeout/10 + 1)).subscribe()
```

Comment:
Why is this division by ten peformed? According to the [documentation](https://cherrypy.readthedocs.io/en/3.2.6/refman/process/plugins/index.html), frequency expects seconds, so I'm not entirely sure why we convert the value passed via the envvar here.